pkg/controllers/cloud/cloudprovideraccount_controller_test.go

```diff
@@ -148,5 +149,53 @@ var _ = Describe("CloudProviderAccount Controller", func() {
 			err = reconciler.processDelete(&testAccountNamespacedName)
 			Expect(err).Should(HaveOccurred())
 		})
+		It("Secret watcher", func() {
+			var err error
+			ch := make(chan error)
+			reconciler.clientset = fakewatch.NewSimpleClientset()
+			credential := `{"accessKeyId": "keyId","accessKeySecret": "Secret"}`
+			err = os.Setenv("POD_NAMESPACE", testSecretNamespacedName.Namespace)
+			Expect(err).ShouldNot(HaveOccurred())
+			// Create secret
+			_, err = reconciler.clientset.CoreV1().Secrets(testSecretNamespacedName.Namespace).Create(context.Background(), secret, v1.CreateOptions{})
+			Expect(err).ShouldNot(HaveOccurred())
+			_ = fakeClient.Create(context.Background(), secret)
+			// Create CPA
+			_ = fakeClient.Create(context.Background(), account)
+			go func() {
+				//err = reconciler.setupSecretWatcher()
+				//Expect(err).Should(HaveOccurred())
```

Comment:
nit: remove?

Cleanup the unit-tests and remove unused code.

Reply:
Solved.